.env.example

```diff
@@ -0,0 +1,8 @@
+# Middleware Environment Variables
+
+# Database Configuration
+DB_URL=postgresql://postgres:QAoA62tckKE68oCgTcWC@containers-us-west-154.railway.app:5680/railway
+
+
+# Server Configuration
+PORT = 3003
```

Comment:
It would be better if you didn't shared this database connection here, remember this repo is a public repo.

this file should have been 
```suggestion
# Middleware Environment Variables

# Database Configuration
DB_URL=


# Server Configuration
PORT =
```

Reply:
noted